android/build.gradle

```diff
@@ -2,11 +2,11 @@
 buildscript {
     ext.versions = [
       'java'               : JavaVersion.VERSION_11,
-      'androidGradlePlugin': '7.2.1',
+      'androidGradlePlugin': '7.4.2',
```

Comment:
Should this be 8.0.2?

Reply:
no, I ran into some issues trying to upgrade it to 8.0.2 and since the goal here was not to do an upgrade, but to fix an issue that showed up on newer devices that were not covered using the 7.2.x version of AGP.

Reply:
makes sense, thanks.